extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergTableWriter.java

```diff
@@ -475,6 +546,16 @@ private List<CompletedParquetWrite> writeParquet(
             } else {
                 newDataLocation = getDataLocation();
             }
+
+            if (!sortColumnNames.isEmpty()) {
+                try {
+                    dhTable = dhTable.sort(sortColumnNames);
```

Comment:
I missed this in an earlier review, but we should probably wrap all of these computations under a liveness scope; we should make sure the new tables we create with `updateView` / `sort` are contained within a temporary liveness scope,  `LivenessScopeStack.open()`.